jsk_baxter_robot/jsk_baxter_startup/baxter.launch

```diff
@@ -17,6 +17,8 @@
   <arg name="launch_robot_state_publisher" default="false"/>
   <arg name="left_electric_gripper" default="false"/>
   <arg name="right_electric_gripper" default="false"/>
+  <arg name="arm_interpolation" default="minjerk" />
+  <arg name="arm_control_mode" default="position_w_id" />
```

Comment:
please consider adding `doc=` to `<arg` tag.
http://wiki.ros.org/roslaunch/XML/arg

Reply:
I updated to add `doc`